examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "UO_hAhMx0rwv",
+        "colab_type": "code",
+        "outputId": "493bd23a-d07a-46db-e634-d38a09f70ef3",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 119
+        }
+      },
+      "source": [
+        "import os\n",
+        "print(os.getcwd())\n",
+        "data_dir = 'data'\n",
+        "!mkdir $data_dir\n",
+        "import glob\n",
+        "import subprocess\n",
+        "import tarfile\n",
+        "import wget\n",
+        "\n",
+        "# Download the dataset. This will take a few moments...\n",
+        "print(\"******\")\n",
+        "if not os.path.exists(data_dir + '/an4_sphere.tar.gz'):\n",
+        "    an4_url = 'http://www.speech.cs.cmu.edu/databases/an4/an4_sphere.tar.gz'\n",
+        "    an4_path = wget.download(an4_url, data_dir)\n",
+        "    print(f\"Dataset downloaded at: {an4_path}\")\n",
+        "else:\n",
+        "    print(\"Tarfile already exists.\")\n",
+        "    an4_path = data_dir + '/an4_sphere.tar.gz'\n",
+        "\n",
+        "# Untar and convert .sph to .wav (using sox)\n",
+        "tar = tarfile.open(an4_path)\n",
+        "tar.extractall(path=data_dir)\n",
+        "\n",
+        "print(\"Converting .sph to .wav...\")\n",
+        "sph_list = glob.glob(data_dir + '/an4/**/*.sph', recursive=True)\n",
+        "for sph_path in sph_list:\n",
+        "    wav_path = sph_path[:-4] + '.wav'\n",
+        "    cmd = [\"sox\", sph_path, wav_path]\n",
+        "    subprocess.run(cmd)\n",
+        "print(\"Finished conversion.\\n******\")"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "/content\n",
+            "******\n",
+            "Dataset downloaded at: data/an4_sphere.tar.gz\n",
+            "Converting .sph to .wav...\n",
+            "Finished conversion.\n",
+            "******\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LEKDkOSimsKr",
+        "colab_type": "text"
+      },
+      "source": [
+        "Since an4 is not designed for speaker recognition, this facilitates the oppurtunity to demostrate how you can generate manifest files that are necessary for training. These methods can be applied to any dataset to get similar training manifest files. \n",
+        "\n",
+        "First get a scp file(s) which has all the wav files with absolute path for each of train,dev and test set. This can be easily done by `find` bash command"
```

Comment:
`find` bash command" -> the `find` bash command"

Reply:
Done